assets/src/stories-editor/components/inserter/index.js

```diff
@@ -112,7 +112,7 @@ class Inserter extends Component {
 				position={ position }
 				onToggle={ this.onToggle }
 				expandOnMobile
-				headerTitle={ __( 'Add a block', 'amp' ) }
+				headerTitle={ __( 'Add block', 'amp' ) }
```

Comment:
I'm not an English expert at all but wouldn't `Add block` be used if it's for a specific block and `Add a block` more accurate here?

Reply:
Actually, I think we should use `element` instead of `block` here anyway.

Mainly removed the `a` to consolidate strings.

Reply:
Yeah, this would probably be fine with or without `a`.